tests-utils/src/main/scala/tech.beshu.ror.utils/containers/DockerfileBuilderOps.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.utils.containers
+
+import org.testcontainers.images.builder.dockerfile.DockerfileBuilder
+
+object DockerfileBuilderOps {
+
+  implicit class OptionalCommandRunner(val builder: DockerfileBuilder) extends AnyVal {
+
+    def runWhen(condition: Boolean, command: String): DockerfileBuilder = {
```

Comment:
maybe 2 parameters list will make the DSL nicer:

```
[...]
  .runWhen(isSthEnabled) {
    // comand
  }
```
?

Reply:
IMO current approach looks better: 
```scala
.run("...")
.runWhen(internodeSslEnabled, "...")
.run("...")
```
than: 
```scala
.run("...")
.runWhen(internodeSslEnabled) {
 "..."
}
.run("...")
```